### PR TITLE
Proposed content of pathogenesis/transmission manuscript

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

This PR proposes the text that will be one of the manuscripts, focusing on pathogenesis and transmission. The introduction and discussion are new text, other edits were just cleaning up the existing text.

This also moves a bit of text out of this paper and into the evolution paper, although I am definitely open to ideas on whether that's a good change.

Feedback on the conclusions would be extra helpful -- this is dense and I imagine there are other points we could try to drill home.

Current word count: 12,032 (including headings, citations, etc.)

### Related issues

#744 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by dziakj1 — APPROVED** [review]

**Review by nilswellhausen — APPROVED** [review]

File: content/07.pathogenesis.md [hunk 1]
```diff
@@ -438,94 +419,42 @@ Several studies used phylogenetic analysis to determine the source of local COVI
...
+Critical insights into how the virus interacts with human cells have been gained from previous research in HCoV and other viral infections.
 As with other HCoV, the immune response to SARS-CoV-2 is likely driven by detection of its spike protein, which allows it to enter cells through the ACE2 receptor.
 Epithelial cells have also emerged as the major cellular target of the virus, contextualizing the respiratory and gastrointestinal symptoms that are frequently observed in COVID-19.
+Many of the mechanisms that facilitate the pathogenesis of SARS-CoV-2 are currently under consideration as possible targets for the treatment or prevention of COVID-19 [cite therapeutics].
+Research in other viruses also provides a foundation for understanding the transmission of SARS-CoV-2 among people and can therefore inform efforts to control the virus's spread.
+While the extent to which aerosol and fomite transmission play a role in the spread of SARS-CoV-2, in general this virus seems to be spread primarily by droplet transmission, much like SARS-CoV-1 and MERS-CoV.
```
Comment (RLordan) [inline content/07.pathogenesis.md:430]: I was reading the transmission section and this conclusion. I think there is a lot less emphasis these days on fomite transmission and much more emphasis on transmission by respiratory droplets within close contact. Certainly fomite transmission does occur, but the CDC does not acknowledge it as the main route. I would have edited, but I don’t know who wrote this section so they may want to review it themselves.

https://www.cdc.gov/coronavirus/2019-ncov/more/scientific-brief-sars-cov-2.html
https://www.cdc.gov/coronavirus/2019-ncov/faq.html#Spread

File: content/07.pathogenesis.md [hunk 2]
```diff
@@ -438,94 +419,42 @@ Several studies used phylogenetic analysis to determine the source of local COVI
...
+While many efforts have focused on adults, and especially older adults because of the susceptibility of this demographic, additional research is needed to understand the presentation of COVID-19 and MIS-C in pediatric patients.
+As more information is uncovered about the pathogenesis of HCoV and SARS-CoV-2 specifically, the diverse symptomology of COVID-19 has and likely will continue to conform with the ever-broadening understanding of how SARS-CoV-2 functions within a human host.
+
+While the SARS-CoV-2 virus is very similar to other HCoV in several ways, including in its genomic structure and the structure of the virus itself, there are also some differences that may account for differences in the COVID-19 pandemic compared to the SARS and MERS of the past two decades.
+The R~0~ of SARS-CoV-2 has been estimated to be similar to SARS-CoV-1 but much higher than that of MERS-CoV [@doi:10.1093/ajcp/aqaa029; @doi:10.1093/ajcp/aqaa029]. 
+While the structure of the viruses are very similar, evolution among these species may account for differences in their transmissibility and virulence.
```
Comment (RLordan) [inline content/07.pathogenesis.md:445]:
```suggestion
While the structures of the viruses are very similar, evolution among these species may account for differences in their transmissibility and virulence.
```

**Review by RLordan — CHANGES_REQUESTED** [review]
Hey @rando2 some suggested edits here. The transmission bit is probably the most important bit to clarify. Thanks a million!

File: content/40.inequality.md [hunk 3]
```diff
@@ -9,6 +9,93 @@ The fact that differences in survival between Black and white patients were no l
...
+Statistics from a number of cities indicate significant discrepancies between the proportion of COVID-19 cases and deaths in Black Americans relative to their representation in the general population [@doi:10/ggv494].
+In addition to Black Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic Americans and in Native American and Alaskan Native communities, including the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055; @url:https://journalhosting.ucalgary.ca/index.php/jisd/article/view/70753/54416].
+In Brazil indigenous communities likewise carry an increased burden of COVID-19 [@doi:10.1126/science.abc0073].
```
Comment (ajlee21) [inline content/40.inequality.md:71]:
```suggestion
In Brazil, indigenous communities likewise carry an increased burden of COVID-19 [@doi:10.1126/science.abc0073].
```

File: content/40.inequality.md [hunk 4]
```diff
@@ -9,6 +9,93 @@ The fact that differences in survival between Black and white patients were no l
...
+For example, some evidence suggests that female sex hormones may be protective [@doi:10.1186/s13293-020-00304-9; @doi:10.1371/journal.ppat.1008570].
+ACE2 expression in the kidneys of male mice was observed to be twice as high as that of females, and a regulatory effect of estradiol on ACE2 expression was demonstrated by removing the gonads and then supplementing with estradiol [@doi:10.1186/2042-6410-1-6; @doi:10.1371/journal.ppat.1008570].
+Other work in mice has shown an inverse association between mortality due to SARS-CoV-1 and estradiol, suggesting a protective role for the sex hormone [@doi:10.1371/journal.ppat.1008570].
+Similarly, evidence suggests that similar patterns might be found in other tissues: preliminary analysis suggested that male patients with aortic valve stenosis may show higher levels of ACE2 expression in the myocardium than females, although this pattern was not found in controls, and research has indicated that females may respond to lower doses of heart medications that act on a pathway shared with ACE2 (Renin angiotensin aldosterone system or RAAS) [@doi:10.1186/s13293-020-00304-9].
```
Comment (ajlee21) [inline content/40.inequality.md:43]:
```suggestion
Similarly, evidence suggests that similar patterns might be found in other tissues: preliminary analysis suggested that male patients with aortic valve stenosis showed higher levels of ACE2 expression in the myocardium than females, although this pattern was not found in controls, and research indicated that females responded to lower doses of heart medications that act on a pathway shared with ACE2 (Renin angiotensin aldosterone system or RAAS) [@doi:10.1186/s13293-020-00304-9].
```

Comment (ajlee21) [reply content/40.inequality.md:43]: Not sure what the wording meant by "suggested" here. I wonder if it should just say `Similarly, evidence suggested that similar patterns might be found in other tissues: preliminary analysis showed that male patients with aortic valve stenosis had higher levels...`

**Review by ajlee21 — APPROVED** [review]
This section looks great! Just had some very minor comments

File: content/40.inequality.md [hunk 5]
```diff
@@ -9,6 +9,93 @@ The fact that differences in survival between Black and white patients were no l
...
+This variation may be influenced by demographic (e.g., average age in the area), medical (e.g., the prevalence of comorbid conditions such as diabetes), and social (e.g., income or healthcare availability) factors that vary geographically.
+Additionally, some of the same factors may influence an individual's probability of exposure to SARS-CoV-2, their risk of developing a more serious case of COVID-19 that would require hospitalization, and their access to medical support.
+As a result, quantifying or comparing susceptibility among individuals, communities, or other groups requires consideration of a number of complex phenomena that intersect across many disciplines of research.
+In this section, the term "risk factors" is used to refer to variables that are statistically associated with more severe COVID-19 outcomes.
+Some are intrinsic characteristics that have been observed to carry an association with variation in outcomes, whereas others may be more functionally linked to the pathophysiology of COVID-19.
+
+#### Patient Traits Associated with Increased Risk
+
+Two traits that have been consistently associated with more severe COVID-19 outcomes are male sex and advanced age.
```
[the remark this answers is not in the document]
Comment (rando2) [reply content/40.inequality.md:27]:
```suggestion
Two traits that have been consistently associated with more severe COVID-19 outcomes are male sex and advanced age (typically defined as 60 or older, with the greatest risk among those 85 and older [@url:https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/older-adults.html]).
```

File: content/07.pathogenesis.md [hunk 6]
```diff
@@ -438,94 +419,41 @@ Several studies used phylogenetic analysis to determine the source of local COVI
...
+The R~0~ of SARS-CoV-2 has been estimated to be similar to SARS-CoV-1 but much higher than that of MERS-CoV [@doi:10.1093/ajcp/aqaa029; @doi:10.1093/ajcp/aqaa029]. 
+While the structures of the viruses are very similar, evolution among these species may account for differences in their transmissibility and virulence.
+For example, the acquisition of a furin cleavage site the S1/S2 boundary within the SARS-CoV-2 S protein may be associated with increased virulence.
+Additionally, concerns have been raised about the accumulation of mutations within the SARS-CoV-2 species itself, and whether these could influence virulence [cite evolution].
+The coming of age of genomic technologies has made these types of analyses feasible, and genomics research that characterize changes in SARS-CoV-2 along with temporal and spatial movement is likely to provide additional insights into whether within-species evolution influences the effect of the virus on the human host.
```
Comment (rando2) [inline content/07.pathogenesis.md:448]:
```suggestion
The coming of age of genomic technologies has made these types of analyses feasible, and genomics research characterizing changes in SARS-CoV-2 along with temporal and spatial movement is likely to provide additional insights into whether within-species evolution influences the effect of the virus on the human host.
```
Minor change to @ajlee21's text suggestion

File: content/40.inequality.md [hunk 7]
```diff
@@ -9,6 +9,93 @@ The fact that differences in survival between Black and white patients were no l
...
+For example, some evidence suggests that female sex hormones may be protective [@doi:10.1186/s13293-020-00304-9; @doi:10.1371/journal.ppat.1008570].
+ACE2 expression in the kidneys of male mice was observed to be twice as high as that of females, and a regulatory effect of estradiol on ACE2 expression was demonstrated by removing the gonads and then supplementing with estradiol [@doi:10.1186/2042-6410-1-6; @doi:10.1371/journal.ppat.1008570].
+Other work in mice has shown an inverse association between mortality due to SARS-CoV-1 and estradiol, suggesting a protective role for the sex hormone [@doi:10.1371/journal.ppat.1008570].
+Similarly, evidence suggests that similar patterns might be found in other tissues: preliminary analysis suggested that male patients with aortic valve stenosis showed higher levels of ACE2 expression in the myocardium than females, although this pattern was not found in controls, and research indicated that females responded to lower doses of heart medications that act on a pathway shared with ACE2 (Renin angiotensin aldosterone system or RAAS) [@doi:10.1186/s13293-020-00304-9].
```
Comment (rando2) [inline content/40.inequality.md:43]:
```suggestion
Similarly, evidence suggests that similar patterns might be found in other tissues.
A preliminary analysis identified higher levels of ACE2 expression in the myocardium of male patients with aortic valve stenosis showed  than female patients, although this pattern was not found in controls [@doi:10.1186/s13293-020-00304-9].
Additionally, research has indicated that females respond to lower doses than males of heart medications that act on the Renin angiotensin aldosterone system (RAAS) pathway, which is shared with ACE2 [@doi:10.1186/s13293-020-00304-9].
```

@ajlee21 does this help?

**Review by rando2 — COMMENTED** [review]
Thank you so much @RLordan and @ajlee21 for all the feedback!

File: content/07.pathogenesis.md [hunk 8]
```diff
@@ -178,17 +170,17 @@ IL-6 is a pleiotropic cytokine that plays an integral role in both the inflammat
...
 However, the argument has been made that while the cytokine levels observed in COVID-19 patients fall outside of the normal range, they are not be as high as typically found in patients with ARDS [@doi:10.1001/jamainternmed.2020.3313].
-Regardless, inflammation has received significant interest both in regards to the pathology of COVID-19 as well as potential avenues for treatment, as the similarities between the cytokine storm and the pathophysiology of COVID-19 has led to the suggestion that a number of immunomodulatory pharmaceutical interventions could hold therapeutic value for the treatment of COVID-19 [@doi:10.1016/j.jaut.2020.102452].
+Regardless, inflammation has received significant interest both in regards to the pathology of COVID-19 as well as potential avenues for treatment, as the similarities between the cytokine storm and the pathophysiology of COVID-19 has led to the suggestion that a number of immunomodulatory pharmaceutical interventions could hold therapeutic value for the treatment of COVID-19 [@doi:10.1016/j.jaut.2020.102452] [cite therapeutics section].
 
 #### Pediatric Presentation
```
[the remark this answers is not in the document]
Comment (RLordan) [reply content/07.pathogenesis.md:185]: Done @rando2 , let me know if you want more detail :)

File: content/07.pathogenesis.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (RLordan) [reply content/07.pathogenesis.md:438]: This is perfect @rando2

**Comment by rando2** [comment]
@ajlee21 and @RLordan I just wanted to check and see if these changes were sufficient to address your concerns! @RLordan if you are happy with them and don't mind changing your review, I can try to get this merged in. Thank you both again!

**Comment by RLordan** [comment]
> @ajlee21 and @RLordan I just wanted to check and see if these changes were sufficient to address your concerns! @RLordan if you are happy with them and don't mind changing your review, I can try to get this merged in. Thank you both again!

Yes, super @rando2

**Review by RLordan — APPROVED** [review]

**Comment by rando2** [comment]
Awesome, thanks @RLordan!

File: content/07.pathogenesis.md [hunk 9]
```diff
@@ -48,7 +40,7 @@ Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 to
...
 Genome organization is highly conserved within the order [@doi:10.1007/978-1-4939-2438-7_1].
 There are three major regions: one containing the replicase gene and one containing the genes encoding structural proteins [@doi:10.1007/978-1-4939-2438-7_1].
-The replicase gene comprises about two-thirds of the genome of coronaviruses and consists of two open reading frames that are translated with ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
+The replicase gene comprises about two-thirds of the genome and consists of two open reading frames that are translated with ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
 This polypeptide is then translated into 16 non-structural proteins (nsp), except in Gammacoronaviruses where nsp1 is absent, that form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
 The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
 Additional accessory genes are sometimes present between these two regions, depending on the species or strain. <!-- TODO: Add a figure showing the structure of the genome.-->
```
[the remark this answers is not in the document]
Comment (ajlee21) [reply content/07.pathogenesis.md:45]: I think visuals are always very helpful. But I wouldn't say its urgent given that there definitely exists many online for people to google. I can try to work on making one but I don't think it needs to hold up the submission

**Comment by ajlee21** [comment]
Looks good to me too. Added a response about the drawing

File: content/07.pathogenesis.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/07.pathogenesis.md:45]: Thank you so much @ajlee21! @nilswellhausen is also incredibly talented at scientific visualization and may have thoughts!

[the remark this answers is not in the document]
Comment (nilswellhausen) [reply content/07.pathogenesis.md:45]: Happy to put a figure together! Are you referring to the entire pathogenesis section or a figure for a specific part ? Either way, if someone wants to send me a basic drawing (can be pen and paper style) I can make it into a figure in less than 2 hours.

[the remark this answers is not in the document]
Comment (ajlee21) [reply content/07.pathogenesis.md:45]: Amazing. I think this is a figure illustrating the structure of the CoV virion particle. So I was thinking something very basic like this: https://www.google.com/search?q=covid+structure&sxsrf=ALeKk02TWfkwENRf8nrvexytwQfQvuDy8A:1610588734617&tbm=isch&source=iu&ictx=1&fir=Pr35tXoGibc1uM%252C5oS0v-dQvEwIJM%252C_&vet=1&usg=AI4_-kRs2tvodunZgUxwkX-D9EGtwckM6A&sa=X&ved=2ahUKEwiEk5_LppruAhUMGVkFHYvpDRMQ9QF6BAgYEAE&biw=2076&bih=1456#imgrc=Pr35tXoGibc1uM

File: content/07.pathogenesis.md [hunk 10]
```diff
@@ -4,42 +4,34 @@
...
+Additionally, the fundamental biology of the virus can provide insights into how it is transmitted among people, which are in turn critical to efforts to control its spread.
+As a result, a thorough understanding of the pathogenesis of SARS-CoV-2 is a critical foundation on which to build an understanding of COVID-19 and the pandemic as a whole.
+
+The rapid identification and release of the genomic sequence of the virus in January 2020 [@doi:10.46234/ccdcw2020.017] provided early insight into the virus in a comparative genomic context.
+The viral genomic sequence clusters with known coronaviruses (order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*).
+Phylogenetic analysis of the coronaviruses reveals four major subclades, each corresponding to a genus: the alpha, beta, delta and gamma coronaviruses.
+Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species, and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
```
[the remark this answers is not in the document]
Comment (nilswellhausen) [reply content/07.pathogenesis.md]: Untitled (7)

[the remark this answers is not in the document]
Comment (nilswellhausen) [reply content/07.pathogenesis.md]: Let me know what you think!

[the remark this answers is not in the document]
Comment (RLordan) [reply content/07.pathogenesis.md]: Awesome @nilswellhausen

[the remark this answers is not in the document]
Comment (ajlee21) [reply content/07.pathogenesis.md]: That's beautiful! Way better than anything i could've made:) Thank you

[the remark this answers is not in the document]
Comment (rando2) [reply content/07.pathogenesis.md]: WOW! This is incredible!!!

**Pull request merged**